lua/darkrp_modules/levels/sh_config.lua

```diff
@@ -2,81 +2,213 @@
 // Configuration file  //
 /////////////////////////
 
-LevelSystemConfiguration = {} // Ignore
-Printers = {} // Ignore
+LevelSystemConfiguration = {} -- Ignore
+local Printers = {} -- Ignore
+local Books = {} -- Ignore
 
+//Language settings
+LevelSystemConfiguration.Language = "EN" -- (available: FR, EN, PL)
```

Comment:
Nice - but this isn't the correct way to do languages. DarkRP has DarkRP.getPhrase and DarkRP.addLanguage
See https://github.com/vrondakis/syndicate/blob/master/gamemode/languages/english.lua for example. Message me if you need help